DynamicModeler/Logic/vtkSlicerDynamicModelerExtrudeTool.cxx

```diff
@@ -104,32 +108,45 @@ vtkSlicerDynamicModelerExtrudeTool::vtkSlicerDynamicModelerExtrudeTool()
   /////////
   // Parameters
 
-  ParameterInfo parameterExtrusionScale(
-    "Extrusion scale",
-    "Length of normals, line or point distance is scaled by this value to get the offset surface position. Use negative value for reversing extrusion direction.",
-    "ExtrusionScale",
+  ParameterInfo parameterLengthMode(
+    "Length mode",
+    "Can be fixed length or scaled length. Scaled mode is ignored for planes",
```

Comment:
```suggestion
    "If absolute then the extrusion length is set to 'Extrusion length' parameter value. If relative then the length defined by the input markup will be multiplied by the `Extrusion length` parameter value to compute the extrusion length.,
```

Reply:
I don't think we need to discuss the special case for planes. Plane normals just don't have length, but the direction vector always has unit length.